utils/generate_types/overrides.d.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Protocol} from './protocol';
+import { ChildProcess } from 'child_process';
+import { EventEmitter } from 'events';
+/**
+ * Can be converted to JSON
+ */
+type Serializable = {}
+type ConnectionTransport = {}
+
+type Boxed<Args extends any[]> = { [Index in keyof Args]: Args[Index] | JSHandle<Args[Index]> };
+type PageFunction<Args extends any[], R = any> = string | ((...args: Args) => R | Promise<R>);
+type PageFunctionOn<On, Args extends any[], R = any> = string | ((on: On, ...args: Args) => R | Promise<R>);
+
+type Handle<T> = T extends Node ? ElementHandle<T> : JSHandle<T>;
+type ElementHandleForTag<K extends keyof HTMLElementTagNameMap> = ElementHandle<HTMLElementTagNameMap[K]>;
+
+type WaitForSelectorOptionsNotHidden = PageWaitForSelectorOptions & {
+  visibility: 'visible'|'any';
```

Comment:
This has been changed recently.

Reply:
Unfortunately my tests for this didn't catch it somehow. I don't know how to make them more strict, but I fixed it.

Reply:
Done